.gitmodules

```diff
@@ -4,7 +4,7 @@
 [submodule "overcooked_ai"]
 	path = overcooked_ai
 	url = git@github.com:HumanCompatibleAI/overcooked_ai.git
-	branch = overcooked_ai_improvements
+	branch = new_mdp_layouts
```

Comment:
Probably should be changed to master, assuming that PR is going to be merged soon

Reply:
Ah, just say you already addressed this in the description – makes sense, i don't think it's urgent to change this

Reply:
went ahead and changed it to master